.github/workflows/release-suite-native-preview-update.yml

```diff
@@ -0,0 +1,49 @@
+name: "[Release] suite-native preview update"
+
+on:
+  push:
+    branches: [develop]
+  pull_request:
+    paths:
+      - "suite-native/**"
+      - "suite-common/**"
+  workflow_dispatch:
+
+  jobs:
+    update:
+      if: github.repository == 'trezor/trezor-suite' || github.repository == 'trezor/trezor-suite-private'
+    name: EAS Update
+    environment: develop-suite-native
```

Comment:
I guess this is not correct @vdovhanych?

Reply:
The environment is only available from protected branches. I can create an environment that is accessible from all branches, but for that, let's do a different org in the expo so we do not expose the API key to all of the branches.